src/v2/guide/conditional.md

```diff
@@ -170,30 +170,30 @@ new Vue({
 </script>
 {% endraw %}
 
-Note that the `<label>` elements are still efficiently re-used, because they don't have `key` attributes.
+Remarquez que les éléments `<label>` sont encore réutilisés efficacement, car ils n'ont pas d'attributs `key`.
 
 ## `v-show`
 
-Another option for conditionally displaying an element is the `v-show` directive. The usage is largely the same:
+Une autre option pour afficher conditionnellement un élément est la directive `v-show`. L'utilisation est en grande partie la même :
 
 ``` html
-<h1 v-show="ok">Hello!</h1>
+<h1 v-show="ok">Bonjour !</h1>
 ```
 
-The difference is that an element with `v-show` will always be rendered and remain in the DOM; `v-show` simply toggles the `display` CSS property of the element.
+La différence est qu'un élément avec `v-show` sera toujours restitué et restera dans le DOM ; `v-show` permute simplement la propriété CSS `display` de l'élément.
 
-<p class="tip">Note that `v-show` doesn't support the `<template>` syntax, nor does it work with `v-else`.</p>
+<p class="tip">Notez que `v-show` ne prend pas en charge la syntaxe de `<template>` et ne fonctionne pas avec `v-else`.</p>
 
 ## `v-if` vs `v-show`
 
-`v-if` is "real" conditional rendering because it ensures that event listeners and child components inside the conditional block are properly destroyed and re-created during toggles.
+`v-if` est un « vrai » rendu conditionnel car il garantit que les écouteurs d'événements et les composants enfants à l'intérieur de la structure conditionnelle soient correctement détruits et recréés lors des permutations.
```

Comment:
oui, un garde du corps est cool car il garantit que ma sécurité **est** assurée. 
Pour les écouteurs d'évènements, on évoque un fait absolument certain et inévitable de programmation : ils sont détruits ( et en plus c'est garanti :D )

Ca te donne l'impression de sonner bizarre car dans le langage courant on entend souvent le subjonctif à la place de l'indicatif (par exemple on dit souvent "après que je sois parti" alors que c'est bien "après que je suis parti").